packages/expo-camera/src/ExpoCamera.web.tsx

```diff
@@ -48,17 +44,17 @@ const ExponentCamera = forwardRef(
 
     const isQRScannerEnabled = useMemo<boolean>(() => {
       return Boolean(
-        props.barCodeScannerSettings?.barCodeTypes?.includes('qr') && !!props.onBarCodeScanned
+        props.barcodeScannerSettings?.barcodeTypes?.includes('qr') && !!props.onBarcodeScanned
       );
-    }, [props.barCodeScannerSettings?.barCodeTypes, props.onBarCodeScanned]);
+    }, [props.barcodeScannerSettings?.barcodeTypes, props.onBarcodeScanned]);
 
     useWebQRScanner(video, {
-      interval: props.barCodeScannerSettings?.interval,
+      interval: 300,
```

Comment:
Do we need to hardcode this interval? What about other platforms? If we don't have option `interval` in the `barCodeScannerSettings `, we can add it.

Reply:
@lukmccall, I certainly agree that it would be preferable if it wasn't a hard coded value. In the legacy package it's there, but in the latest version of the camera it's not covered in the types nor the docs. I'm not sure why it's removed in the first place, but figured this must have been a made decision.

I have not checked, but as it's not covered in nor types and docs I have just assumed it was removed in all platforms  accordingly.

Reply:
That options seems to be available in the legacy version:
https://github.com/expo/expo/blob/3e3b8a6fea89dcb835437601f5e444f5103b7b30/packages/expo-camera/src/legacy/Camera.types.ts#L544
but not in the new one:
https://github.com/expo/expo/blob/3e3b8a6fea89dcb835437601f5e444f5103b7b30/packages/expo-camera/src/Camera.types.ts#L468

I believe we can merge it, and later we can introduce the interval property 🤔 cc @alanjhughes

Reply:
Exactly - in the legacy version it's defined. But in the current version it's not. 

In the meantime I decided to give it a number (this may need to be adjusted), rather than undefined, so that the interval will actually run when intending to scan. Making the change in this file will not ruin the legacy version (that has `interval`-support), and it will at least be somewhat functional with the new camera.

Again, spontaneously I agree that it would be better with a value that is not hardcoded but I have assumed that there probably is probably some reasoning behind it's removal.

Reply:
Whenever you have time, I'm really excited to hear your final thoughts and get your take on this PR @alanjhughes 😀